### PR TITLE
Fixing mailmap issue

### LINKED ISSUE #60: update-lookup: crashes with "ValueError: too many values to unpack" on .mailmap parsin
```shell
(git)lena:~datalad/datalad-master[master]git
$> tributors update-lookup
INFO:allcontrib:Updating .tributors cache from .all-contributorsrc
INFO:    zenodo:Updating .tributors cache from .zenodo.json
INFO:   mailmap:Updating .tributors cache from .mailmap
Traceback (most recent call last):
  File "/home/yoh/proj/tributors/venvs/dev3/bin/tributors", line 33, in <module>
    sys.exit(load_entry_point('tributors', 'console_scripts', 'tributors')())
  File "/home/yoh/proj/tributors/tributors/client/__init__.py", line 175, in main
    main(args, extra)
  File "/home/yoh/proj/tributors/tributors/client/lookup.py", line 52, in main
    client.update_resource(resources=resources, params=extra)
  File "/home/yoh/proj/tributors/tributors/main/__init__.py", line 97, in update_resource
    resource.update_lookup()
  File "/home/yoh/proj/tributors/tributors/main/parsers/mailmap.py", line 63, in update_lookup
    self.load_data()
  File "/home/yoh/proj/tributors/tributors/main/parsers/mailmap.py", line 41, in load_data
    name, email = line.split("<")
ValueError: too many values to unpack (expected 2)
(dev3) 1 15113 ->1.....................................:Wed 18 Aug 2021 09:53:20 AM EDT:.
(git)lena:~datalad/datalad-master[master]git
$> git describe           
0.14.7-544-g379d4e2f9
changes on filesystem:                                                                                                                                                                                               
 .github/workflows/update-contributors.yml | 2 +-
(dev3) 1 15114.....................................:Wed 18 Aug 2021 09:54:31 AM EDT:.
(git)lena:~datalad/datalad-master[master]git
$> tributors --version
0.0.19
changes on filesystem:                                                                                                                                                                                               
 .github/workflows/update-contributors.yml | 2 +-
(dev3) 1 15115.....................................:Wed 18 Aug 2021 09:54:34 AM EDT:.
(git)lena:~datalad/datalad-master[master]git
$> cat .mailmap 
Alejandro de la Vega <delavega@utexas.edu>
Alex Waite <alexqw85@gmail.com> Alex Waite <alex@waite.eu>
Andy Connolly <andrew.c.connolly@dartmouth.edu> <andycon@users.noreply.github.com>
Anisha Keshavan <keshavan@berkeley.edu>
Benjamin Poldrack <benjaminpoldrack@gmail.com> <bpoldrack@users.noreply.github.com>
Christian Mönch <christian.moench@web.de> Christian Moench <christian.monch@conax.com>
Christian Olaf Häusler <der.haeusler@gmx.net> chris <chris@psyinfolaf.ovgu.de>
Dave MacFarlane <driusan@gmail.com> Dave MacFarlane <dave.macfarlane@mcin.ca>
Dave MacFarlane <driusan@gmail.com> Dave MacFarlane <driusan@mcin.ca>
Debanjum Singh Solanky <debanjum@gmail.com> Debanjum <debanjum@gmail.com> debanjum <debanjum@gmail.com>
Gergana Alteva <glalteva@gmail.com> Gergana Alteva <glalteva@users.noreply.github.com>
Jason Gors <jason.gors.work@gmail.com>
John T. Wodder II <git@varonathe.org> <jwodder@users.noreply.github.com>
Kusti Skytén <kusti.skyten@helsinki.fi> <kskyten@users.noreply.github.com>
Michael Hanke <michael.hanke@gmail.com> mih <mih>
Neuroimaging Community <committer@example.com> blah <committer@example.com> 
Neuroimaging Community <committer@example.com> <test@example.com> 
Neuroimaging Community <committer@example.com> unknown <test@nowhere.com> 
Taylor Olson <to.overlake@gmail.com> Taylor Olson <taylor.m.olson.21@dartmouth.edu>
Torsten Stoeter <torsten.stoeter@lin-magdeburg.de>
Vanessa Sochat <vsoch@users.noreply.github.com> 
Yaroslav Halchenko <debian@onerussian.com> 
Yaroslav Halchenko <debian@onerussian.com> <site-github-private@onerussian.com>
Yaroslav Halchenko <debian@onerussian.com> <yoh@smaug.cns.dartmouth.edu>
```

### PR DESCRIPTION
**Pull request by vsoch**
@yarikoptic this should fix #60. The issue was (as I suspected) not fully understanding the mailmap file - it looks like you can have more than one name, email for file.

Since this is just an update-lookup function, it will only serve to find new metadata for existing entries, if I remember correctly!

I also ran black on the code and updated the dates in the headers, sorry for the extra noise!

Signed-off-by: vsoch <vsoch@users.noreply.github.com>

### CODE REVIEW

File: tributors/main/parsers/mailmap.py [hunk 1]
```diff
@@ -38,15 +38,19 @@ def load_data(self):
...
-                name, email = line.split("<")
-                email = email.strip().rstrip(">")
-                self.data[email] = {"name": name.strip()}
+                # mailmap line can have more than one entry, split by right >
+                for entry in line.strip().split(">"):
+                    if not entry:
+                        continue
+                    name, email = entry.split("<")
+                    email = email.strip()
+                    self.data[email] = {"name": name.strip()}
```
Comment (yarikoptic) [inline tributors/main/parsers/mailmap.py]: I believe it should be the `name` of the first entry which is used for all emails you can find later in the line. That is the point of such mailmap entries - to map into another name

Comment (vsoch) [reply tributors/main/parsers/mailmap.py]: For the example you gave me, there were pairs of names and emails (so more than one name per line). You are saying if there isn’t a second name to use the first?

Comment (yarikoptic) [reply tributors/main/parsers/mailmap.py]: Yes
You can run git shortlog -sn to see the effect of the second names collapsing into the record for the first in line

Comment (vsoch) [reply tributors/main/parsers/mailmap.py]: okay gave it a shot (but didn't test yet) - take a look at changes!

**Comment by yarikoptic** [comment]
mailmap issue was fixed in #69. any blacking etc should be redone due to extensive conflicts now.  Please reopen or file a new one

**Pull request closed without merging**